webrtc/videoroom/lib/videoroom_web/templates/room/index.html.eex

```diff
@@ -4,6 +4,5 @@
     </a>
     <h3>Room: <%= @room_id %></h3>
     <div id="videochat" class="VideoChat"> </div>
-    <div id="control" class="Control" />
```

Comment:
I believe that this element is used to display errors coming from websocket. Could be changed to:
```
<div id="videochat-error" class="VideoChatError />
```
and updated respectively in `ui.ts` file.

Reply:
done